networking/metallb/metallb-upgrading-operator.adoc

```diff
@@ -6,18 +6,18 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-The automatic upgrade procedure does not work as expected from {product-title} 4.10 and earlier. A summary of the upgrade procedure is as follows:
+The automatic upgrade procedure does not work as expected from {product-title} 4.10 and earlier. Upgrading from any version greater than {product-title} 4.10 to the latest version works as expected. A summary of the upgrade procedure from {product-title} 4.10 and earlier is as follows:
 
-. Delete the previously installed Operator version for example 4.10. Ensure that the namespace and the `metallb` custom resource are not removed.
+. Delete the 4.10 installed MetalLB Operator version. Ensure that the namespace and the `metallb` custom resource are not removed.
 
-. Install the 4.11 version of the Operator using the CLI. Install the 4.11 version of the Operator in the same namespace that the previously installed Operator version was installed to.
+. Install the {product-version} version of the Operator using the CLI. Install the {product-version} version of the Operator in the same namespace that the previously installed Operator version was installed to.
```

Comment:
If you are going to use want to use the version number to refer to the product, ISG guidance says to use _version <version_number>_ instead of _<version_number> version_.

Reply:
for MetalLB it is tied to OCP version so when OCP rev'd it is rev'd